pyproject.toml

```diff
@@ -15,6 +15,7 @@ PyYAML = "^6.0"
 certifi = "^2022.6.15"
 msgspec = "^0.9.0"
 requests = "^2.28.1"
+jinja2 = "^3.1.2"
```

Comment:
Why is it added as required dependency? Isn't it supposed to as an optional dependency?

Reply:
According to the logic I've implemented, there's no because `import jinja2` there. It's an additional dependency, and it's not part of the standard library.